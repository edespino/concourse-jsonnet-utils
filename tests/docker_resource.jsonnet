local concourse = import 'concourse.libsonnet';

{
  docker: concourse.DockerResource('serverspec-container', 'sirech/dind-ruby', tag = '2.6.3'),
}
