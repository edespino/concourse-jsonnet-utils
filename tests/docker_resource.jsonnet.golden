{
   "docker": {
      "name": "serverspec-container",
      "source": {
         "repository": "sirech/dind-ruby",
         "tag": "2.6.3"
      },
      "type": "docker-image"
   }
}
